machine/install-machine.md

```diff
@@ -80,7 +80,7 @@ done
 ```
 
 Then you need to run `source
-/etc/bash_completion.d/docker-machine-prompt.bash` in your bash
+/etc/bash_completion.d docker-machine-prompt.bash` in your bash
```

Comment:
doesn't look correct to me;

```bash
source /etc/bash_completion.d foo
-bash: source: /etc/bash_completion.d: is a directory
```

can you describe the problem that you're seeing?